components/applied-filters/applied-filters.js

```diff
@@ -103,27 +96,27 @@ class D2lLabsAppliedFilters extends RtlMixin(FilterLocalizeMixin(LitElement)) {
 
 	render() {
 		const filters = this._selectedEntries && this._selectedEntries.length > 0 ?
-			html`<d2l-labs-multi-select-list
-				collapsable
+			html`<d2l-tag-list
+				clearable
 				aria-labelledby="d2l-labs-applied-filters-label"
```

Comment:
This `aria-labelledby` is pointing at an element that doesn't appear to exist? Also `d2l-tag-list` doesn't support such an attribute, and requires a `description` instead.

I guess `d2l-labs-applied-filters`'s `labelText` property should go there? It doesn't appear to be used currently.